minimesos/src/main/java/com/containersol/minimesos/main/CommandUp.java

```diff
@@ -103,7 +94,7 @@ public int getNumAgents() {
         if (numAgents > 0) {
             return numAgents;
         } else {
-            ClusterConfig clusterConfig = getClusterConfig();
+            ClusterConfig clusterConfig = readClusterConfigFromMinimesosFile();
```

Comment:
The single `return` statement is in the middle of the method. Can we restructure the code, so it's actually the last statement?